dom.bs

```diff
@@ -1379,14 +1379,25 @@ an <var>object</var> with <var>event</var>, run these steps:
 
 <h3 id=firing-events>Firing events</h3>
 
-To
-<dfn export id=concept-event-fire lt="fire an event">fire an event named <var>e</var></dfn>
-means that a new <a>event</a> using the
-{{Event}} interface, with its
-{{Event/type}} attribute initialized to
-<var>e</var>, and its {{Event/isTrusted}}
-attribute initialized to <code>true</code>, is to be
-<a>dispatched</a> to the given object.
+<p>To <dfn export id=concept-event-fire lt="fire an event">fire an event</dfn> named <var>e</var> at
+<var>target</var>, optionally given an <var>eventClass</var>, a description of how IDL attributes
```

Comment:
Maybe eventInterface is more accurate? I dunno, I want to use JS terms, but this is kind of Web IDL land. Could go either way.